x-pack/plugins/lens/public/editor_frame_service/editor_frame/data_panel_wrapper.tsx

```diff
@@ -180,53 +180,9 @@ export const DataPanelWrapper = memo((props: DataPanelWrapperProps) => {
     ],
   };
 
-  const [showDatasourceSwitcher, setDatasourceSwitcher] = useState(false);
-
   return (
     <>
       <Easteregg query={externalContext?.query} />
-      {Object.keys(props.datasourceMap).length > 1 && (
```

Comment:
ℹ️ This was adding a popover for changing datasources (in case of more than one). In this PR we are introducing a new datasource so this begun to be displayed but we don't want it. The datasource change is done by the dataview picker.